app/forms/onboarding/request_form.rb

```diff
@@ -49,8 +49,6 @@ def proxy_auth_params
   end
 
   def test_api!
-    return unless ProxyDeploymentService.call(model, v1_compatible: true)
```

Comment:
This is virtually `return unless false` which always returns, therefore I think the correct implementation would be:
```ruby
def test_api!
  return
end
```